CHANGELOG.unreleased.md

```diff
@@ -20,7 +20,7 @@
 - [Home/SR] Fix inability to edit SRs' name [#5057](https://github.com/vatesfr/xen-orchestra/issues/5057) (PR [#5058](https://github.com/vatesfr/xen-orchestra/pull/5058))
 - [Backup] Fix huge logs in case of Continuous Replication or Disaster Recovery errors (PR [#5069](https://github.com/vatesfr/xen-orchestra/pull/5069))
 - [Notification] Fix same notification showing again as unread (PR [#5067](https://github.com/vatesfr/xen-orchestra/pull/5067))
-- [SDN Controller] Fix broken private network creation when specifiyng a preferred center [#5076](https://github.com/vatesfr/xen-orchestra/issues/5076) (PR [#5079](https://github.com/vatesfr/xen-orchestra/pull/5079))
+- [SDN Controller] Fix broken private network creation when specifiyng a preferred center [#5076](https://github.com/vatesfr/xen-orchestra/issues/5076) (PR [#5079](https://github.com/vatesfr/xen-orchestra/pull/5079)) (PR [#5080](https://github.com/vatesfr/xen-orchestra/pull/5080))
```

Comment:
Please resolve conflicts with master.

Reply:
rebased on latest master.
I4ve put 2 PRs on the same entry because both fixes broken private network creation.
Would you prefer I put 2 entries?

Reply:
Nope, that's fine.

```suggestion
- [SDN Controller] Fix broken private network creation when specifiyng a preferred center [#5076](https://github.com/vatesfr/xen-orchestra/issues/5076) (PRs [#5079](https://github.com/vatesfr/xen-orchestra/pull/5079) & [#5080](https://github.com/vatesfr/xen-orchestra/pull/5080))
```